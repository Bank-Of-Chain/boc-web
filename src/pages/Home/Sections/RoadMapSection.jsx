import React from "react";
import { makeStyles } from "@material-ui/core/styles";

// === Components === //
import Timeline from "@components/Timeline";

// === Styles === //
import styles from "./roadMapStyle";

const useStyles = makeStyles(styles);

export default function RoadMapSection() {
  const classes = useStyles();
  const stepOne = [
    {
      date: "March 4, 2022",
      event: "Friend & Family Testing",
    },
    {
      date: "March 4, 2022",
      event: "Logo design",
    },
    {
      date: "March 10, 2022",
      event: "Formulation of token issuance rules",
    },
    {
      date: "April 15, 2022",
      event: "White paper",
    },
    {
      date: "April 15, 2022",
      event: "AWS server leasing and deployment",
    },
  ];
  const stepTwo = [
    {
      date: "May 10, 2022",
      event: "BOC 1.5 Version",
    },
    {
      date: "May 15, 2022",
      event: "Security audit",
    },
    {
      date: "July 15, 2022",
      event: "Leveraging services",
    },
    {
      date: "August 1, 2022",
      event: "BOC 2 Version",
    },
  ];
  return (
    <div className={classes.roadmap}>
      <h3 className={classes.title}>Roadmap.</h3>
      <Timeline events={stepOne} topText="NOW" />
      <Timeline
        events={stepTwo}
        topText="NEXT STEPS"
        firstPlace="left"
        cardColorReverse
        connectorColorRevese
      />
    </div>
  );
}
