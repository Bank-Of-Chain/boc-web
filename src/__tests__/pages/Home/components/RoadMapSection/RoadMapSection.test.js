import React from 'react'
import { render } from '@testing-library/react'
import RoadMapSection from '@/pages/Home/components/RoadMapSection'

test('RoadMapSection Component Render', () => {
  const { asFragment } = render(<RoadMapSection />)
  expect(asFragment()).toMatchSnapshot()
})
