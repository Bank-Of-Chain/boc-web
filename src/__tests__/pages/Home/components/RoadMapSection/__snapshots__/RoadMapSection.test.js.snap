// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoadMapSection Component Render 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-roadmap-1"
  >
    <h3
      class="makeStyles-title-2"
    >
      Roadmap.
    </h3>
    <div
      class="makeStyles-timelineWrapper-3"
    >
      <div
        class="makeStyles-topInfo-4"
      >
        NOW
      </div>
      <div
        class="makeStyles-events-5"
      >
        <div
          class="makeStyles-connector-6"
        />
        <ul
          class="MuiTimeline-root MuiTimeline-alignAlternate makeStyles-timeline-8"
        >
          <li
            class="MuiTimelineItem-root MuiTimelineItem-alignAlternate makeStyles-timelineItem-9"
          >
            <div
              class="MuiTimelineOppositeContent-root MuiTimelineItem-oppositeContent"
            >
              <p
                class="makeStyles-keyTime-11"
              >
                March 4, 2022
              </p>
            </div>
            <div
              class="MuiTimelineSeparator-root makeStyles-separator-12"
            >
              <span
                class="MuiTimelineDot-root makeStyles-dot-13 MuiTimelineDot-defaultGrey"
              />
            </div>
            <div
              class="MuiTimelineContent-root MuiTimelineItem-content"
            >
              <div
                class="makeStyles-timelineCard-14"
              >
                <p
                  class="makeStyles-content-16"
                >
                  Friend & Family Testing
                </p>
                <div
                  class="makeStyles-arrow-17 makeStyles-arrowLeft-18"
                />
              </div>
            </div>
          </li>
          <li
            class="MuiTimelineItem-root MuiTimelineItem-alignAlternate makeStyles-timelineItem-9"
          >
            <div
              class="MuiTimelineOppositeContent-root MuiTimelineItem-oppositeContent"
            >
              <p
                class="makeStyles-keyTime-11"
              >
                March 4, 2022
              </p>
            </div>
            <div
              class="MuiTimelineSeparator-root makeStyles-separator-12"
            >
              <span
                class="MuiTimelineDot-root makeStyles-dot-13 MuiTimelineDot-defaultGrey"
              />
            </div>
            <div
              class="MuiTimelineContent-root MuiTimelineItem-content"
            >
              <div
                class="makeStyles-timelineCard-14"
              >
                <p
                  class="makeStyles-content-16"
                >
                  Logo design
                </p>
                <div
                  class="makeStyles-arrow-17 makeStyles-arrowRight-19"
                />
              </div>
            </div>
          </li>
          <li
            class="MuiTimelineItem-root MuiTimelineItem-alignAlternate makeStyles-timelineItem-9"
          >
            <div
              class="MuiTimelineOppositeContent-root MuiTimelineItem-oppositeContent"
            >
              <p
                class="makeStyles-keyTime-11"
              >
                March 10, 2022
              </p>
            </div>
            <div
              class="MuiTimelineSeparator-root makeStyles-separator-12"
            >
              <span
                class="MuiTimelineDot-root makeStyles-dot-13 MuiTimelineDot-defaultGrey"
              />
            </div>
            <div
              class="MuiTimelineContent-root MuiTimelineItem-content"
            >
              <div
                class="makeStyles-timelineCard-14"
              >
                <p
                  class="makeStyles-content-16"
                >
                  Formulation of token issuance rules
                </p>
                <div
                  class="makeStyles-arrow-17 makeStyles-arrowLeft-18"
                />
              </div>
            </div>
          </li>
          <li
            class="MuiTimelineItem-root MuiTimelineItem-alignAlternate makeStyles-timelineItem-9"
          >
            <div
              class="MuiTimelineOppositeContent-root MuiTimelineItem-oppositeContent"
            >
              <p
                class="makeStyles-keyTime-11"
              >
                April 15, 2022
              </p>
            </div>
            <div
              class="MuiTimelineSeparator-root makeStyles-separator-12"
            >
              <span
                class="MuiTimelineDot-root makeStyles-dot-13 MuiTimelineDot-defaultGrey"
              />
            </div>
            <div
              class="MuiTimelineContent-root MuiTimelineItem-content"
            >
              <div
                class="makeStyles-timelineCard-14"
              >
                <p
                  class="makeStyles-content-16"
                >
                  White paper
                </p>
                <div
                  class="makeStyles-arrow-17 makeStyles-arrowRight-19"
                />
              </div>
            </div>
          </li>
          <li
            class="MuiTimelineItem-root MuiTimelineItem-alignAlternate makeStyles-timelineItem-9"
          >
            <div
              class="MuiTimelineOppositeContent-root MuiTimelineItem-oppositeContent"
            >
              <p
                class="makeStyles-keyTime-11"
              >
                April 15, 2022
              </p>
            </div>
            <div
              class="MuiTimelineSeparator-root makeStyles-separator-12"
            >
              <span
                class="MuiTimelineDot-root makeStyles-dot-13 MuiTimelineDot-defaultGrey"
              />
            </div>
            <div
              class="MuiTimelineContent-root MuiTimelineItem-content"
            >
              <div
                class="makeStyles-timelineCard-14"
              >
                <p
                  class="makeStyles-content-16"
                >
                  AWS server leasing and deployment
                </p>
                <div
                  class="makeStyles-arrow-17 makeStyles-arrowLeft-18"
                />
              </div>
            </div>
          </li>
        </ul>
      </div>
    </div>
    <div
      class="makeStyles-timelineWrapper-3"
    >
      <div
        class="makeStyles-topInfo-4"
      >
        NEXT STEPS
      </div>
      <div
        class="makeStyles-events-5"
      >
        <div
          class="makeStyles-connector-6 makeStyles-connectorColorRevese-7"
        />
        <ul
          class="MuiTimeline-root MuiTimeline-alignAlternate makeStyles-timeline-8"
        >
          <li
            class="MuiTimelineItem-root MuiTimelineItem-alignAlternate makeStyles-timelineItem-9 makeStyles-hidden-10"
          >
            <div
              class="MuiTimelineOppositeContent-root MuiTimelineItem-oppositeContent"
            >
              <p
                class="makeStyles-keyTime-11"
              />
            </div>
            <div
              class="MuiTimelineSeparator-root makeStyles-separator-12"
            >
              <span
                class="MuiTimelineDot-root makeStyles-dot-13 MuiTimelineDot-defaultGrey"
              />
            </div>
            <div
              class="MuiTimelineContent-root MuiTimelineItem-content"
            >
              <div
                class="makeStyles-timelineCard-14 makeStyles-colorReverse-15"
              >
                <p
                  class="makeStyles-content-16"
                >
                  placeholder
                </p>
                <div
                  class="makeStyles-arrow-17 makeStyles-arrowLeft-18"
                />
              </div>
            </div>
          </li>
          <li
            class="MuiTimelineItem-root MuiTimelineItem-alignAlternate makeStyles-timelineItem-9"
          >
            <div
              class="MuiTimelineOppositeContent-root MuiTimelineItem-oppositeContent"
            >
              <p
                class="makeStyles-keyTime-11"
              >
                May 10, 2022
              </p>
            </div>
            <div
              class="MuiTimelineSeparator-root makeStyles-separator-12"
            >
              <span
                class="MuiTimelineDot-root makeStyles-dot-13 MuiTimelineDot-defaultGrey"
              />
            </div>
            <div
              class="MuiTimelineContent-root MuiTimelineItem-content"
            >
              <div
                class="makeStyles-timelineCard-14 makeStyles-colorReverse-15"
              >
                <p
                  class="makeStyles-content-16"
                >
                  BOC 1.5 Version
                </p>
                <div
                  class="makeStyles-arrow-17 makeStyles-arrowRight-19"
                />
              </div>
            </div>
          </li>
          <li
            class="MuiTimelineItem-root MuiTimelineItem-alignAlternate makeStyles-timelineItem-9"
          >
            <div
              class="MuiTimelineOppositeContent-root MuiTimelineItem-oppositeContent"
            >
              <p
                class="makeStyles-keyTime-11"
              >
                May 15, 2022
              </p>
            </div>
            <div
              class="MuiTimelineSeparator-root makeStyles-separator-12"
            >
              <span
                class="MuiTimelineDot-root makeStyles-dot-13 MuiTimelineDot-defaultGrey"
              />
            </div>
            <div
              class="MuiTimelineContent-root MuiTimelineItem-content"
            >
              <div
                class="makeStyles-timelineCard-14 makeStyles-colorReverse-15"
              >
                <p
                  class="makeStyles-content-16"
                >
                  Security audit
                </p>
                <div
                  class="makeStyles-arrow-17 makeStyles-arrowLeft-18"
                />
              </div>
            </div>
          </li>
          <li
            class="MuiTimelineItem-root MuiTimelineItem-alignAlternate makeStyles-timelineItem-9"
          >
            <div
              class="MuiTimelineOppositeContent-root MuiTimelineItem-oppositeContent"
            >
              <p
                class="makeStyles-keyTime-11"
              >
                July 15, 2022
              </p>
            </div>
            <div
              class="MuiTimelineSeparator-root makeStyles-separator-12"
            >
              <span
                class="MuiTimelineDot-root makeStyles-dot-13 MuiTimelineDot-defaultGrey"
              />
            </div>
            <div
              class="MuiTimelineContent-root MuiTimelineItem-content"
            >
              <div
                class="makeStyles-timelineCard-14 makeStyles-colorReverse-15"
              >
                <p
                  class="makeStyles-content-16"
                >
                  Leveraging services
                </p>
                <div
                  class="makeStyles-arrow-17 makeStyles-arrowRight-19"
                />
              </div>
            </div>
          </li>
          <li
            class="MuiTimelineItem-root MuiTimelineItem-alignAlternate makeStyles-timelineItem-9"
          >
            <div
              class="MuiTimelineOppositeContent-root MuiTimelineItem-oppositeContent"
            >
              <p
                class="makeStyles-keyTime-11"
              >
                August 1, 2022
              </p>
            </div>
            <div
              class="MuiTimelineSeparator-root makeStyles-separator-12"
            >
              <span
                class="MuiTimelineDot-root makeStyles-dot-13 MuiTimelineDot-defaultGrey"
              />
            </div>
            <div
              class="MuiTimelineContent-root MuiTimelineItem-content"
            >
              <div
                class="makeStyles-timelineCard-14 makeStyles-colorReverse-15"
              >
                <p
                  class="makeStyles-content-16"
                >
                  BOC 2 Version
                </p>
                <div
                  class="makeStyles-arrow-17 makeStyles-arrowLeft-18"
                />
              </div>
            </div>
          </li>
        </ul>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
